lib/block-supports/duotone.php

```diff
@@ -290,6 +290,59 @@ function gutenberg_tinycolor_string_to_rgb( $color_str ) {
 	}
 }
 
+class WP_Duotone {
+	static function get_id_selector_property_and_maybe_svg( $duotone_attr, $duotone_support ) {
```

Comment:
When there are words like `and` and `maybe` in a function name, it means the function is probably doing too much. And in this case, it feels like that even more so because it generates four outputs, one of which may be `null`.